libsyclinterface/CMakeLists.txt

```diff
@@ -280,7 +280,7 @@ set_target_properties(DPCTLSyclInterface
 
 if (SKBUILD)
     set(_lib_destination dpctl)
-    set(_include_destination dpctl/include/syclinterface)
+    set(_include_destination ${CMAKE_INSTALL_PREFIX}/unused)
```

Comment:
What is this change doing?

Reply:
Avoiding that `syclinterface` header, which are copied to `dpctl/include/syclinterface` as part of build step be overwritten by the install. The install just puts them in `_skbuild/*/cmake-install/unused` folder, and `setuptools` does not copy them into the Python layout, and instead copies files from `dpctl/include/syclinterface` like intended.

Reply:
Then why is this step needed all  together? Will it not work, if the line is completely removed?